### PR TITLE
Added ScaleAutonCommand

### PR DESCRIPTION
**Pull request by PrathamRawat**

### CODE REVIEW

**Review by huiminwu — CHANGES_REQUESTED** [review]
Add logic to get FMS data to a GameData.java file in the util package to avoid redundancies. Refer to PoC FMS branch f 2018-proof-of-concepts.

File: src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java [hunk 1]
```diff
@@ -27,13 +27,13 @@ protected void initialize() {
...
-        Robot.drivetrain.tankDrive((speedMode + 1) * 0.25, (speedMode + 1) * 0.25);
-        isReached = Robot.drivetrain.isOnLine((int) speedMode);
+        Robot.drivetrain.tankDrive((speedNumber + 1) * 0.25, (speedNumber + 1) * 0.25);
+        isReached = Robot.drivetrain.isOnLine((int) speedNumber);
     }
 
     // Make this return true when this Command no longer needs to run execute()
     protected boolean isFinished() {
-        return isReached || (Math.max(Robot.drivetrain.getRightEncoderDistance(),Robot.drivetrain.getLeftEncoderDistance()) > (distance + 5));
+        return isReached || (Math.max(Robot.drivetrain.getRightEncoderDistance(),Robot.drivetrain.getLeftEncoderDistance()) > (distance));
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java:36]: Use Robot.drivetrain.getEncoderDistance() which grabs the max of both sides for you

File: src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java [hunk 2]
```diff
@@ -10,10 +10,10 @@
 public class DrivetrainMoveToLineCommand extends Command {
     private boolean isReached = false; 
     private double distance = 0;
-    private double speedMode = 0;
+    private double speedNumber = 0;// 0,1, or 2, repesents speeds 0.25,0.5,0.75 respectively.
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java]: Do not use this. You're literally converting a speed into an imaginary "speed number" unit and then converting it back to a speed when you use it, which makes it pointless. Just use the raw speed value, don't convert it or anything.

File: src/org/usfirst/frc/team694/robot/commands/auton/SameSideRightAngleAutonCommand.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc.team694.robot.commands.auton;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class SameSideRightAngleAutonCommand extends CommandGroup {
```
Comment (adrisj7) [inline src/org/usfirst/frc/team694/robot/commands/auton/SameSideRightAngleAutonCommand.java]: Don't add this file if there's nothing in it

**Review by TacoTechnica — CHANGES_REQUESTED** [review]
Fix DrivetrainMoveToLineCommand
Remove or populate empty SameSideRightAngleAutonCommand

**Pull request closed without merging**